static/js/components/ProfileProgressControls.js

```diff
@@ -45,7 +46,7 @@ export default class ProfileProgressControls extends React.Component {
         colored
         className="progress-button next"
         onClick={this.saveAndContinue}>
-        <span>{isLastTab ? "I'm Done!" : "Save and Continue"}</span>
```

Comment:
if we're not using `isLastTab` anywhere we should remove all references to it (from the `propTypes` here on `ProfileProgressControls`, but also from the props we pass in).